deezer/client.py

```diff
@@ -145,7 +145,12 @@ def get_object(
         """
         url = self.object_url(object_t, object_id, relation, **kwargs)
         response = self.session.get(url)
-        return self._process_json(response.json(), parent)
+        json = response.json()
+        if "error" in json:
+            raise ValueError(
+                "API request return error for object: %s id: %s" % (object_t, object_id)
+            )
```

Comment:
That would probably do the job for now, but eventually I think this library should raise its own custom exception, all extending from a common base one.